### PR TITLE
new/healthcheck: Added healtcheck api endpoint

### PR DESCRIPTION
**Pull request by CyrilPeponnet**

### CODE REVIEW

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "healthcheck",
    "component": "gaia",
    "pr-id": "504",
    "commit-sha": "f4e7b5a205de203951a54710c92c0d6248cc494b",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "backend",
    "pr-id": "765",
    "commit-sha": "dbd447a54fcd347d5a669a8c6520cb27a93dd7fb",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "underwater",
    "pr-id": "113",
    "commit-sha": "75b116461a71dcb349a2425dea0a481e739a821f",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "healthcheck",
    "component": "gaia",
    "pr-id": "504",
    "commit-sha": "31526a168f7bc8c14af591926ab98968b8e4daa1",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "backend",
    "pr-id": "765",
    "commit-sha": "50fd2672464682f834fc21f861ad19af71740e5f",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "underwater",
    "pr-id": "113",
    "commit-sha": "75b116461a71dcb349a2425dea0a481e739a821f",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "healthcheck",
    "component": "gaia",
    "pr-id": "504",
    "commit-sha": "31526a168f7bc8c14af591926ab98968b8e4daa1",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "backend",
    "pr-id": "765",
    "commit-sha": "d2a2c336683e279de35897328ef544ffb767e6ee",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "underwater",
    "pr-id": "113",
    "commit-sha": "75b116461a71dcb349a2425dea0a481e739a821f",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "healthcheck",
    "component": "backend",
    "pr-id": "765",
    "commit-sha": "b95d15d637f034008d1f1d62946bcccdd21a1c1e",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "underwater",
    "pr-id": "113",
    "commit-sha": "75b116461a71dcb349a2425dea0a481e739a821f",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "gaia",
    "pr-id": "504",
    "commit-sha": "31526a168f7bc8c14af591926ab98968b8e4daa1",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "healthcheck",
    "component": "voila",
    "pr-id": "100",
    "commit-sha": "9e68794449e5ca5e93202d5569eee4726edcf2cb",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "gaia",
    "pr-id": "504",
    "commit-sha": "31526a168f7bc8c14af591926ab98968b8e4daa1",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "backend",
    "pr-id": "765",
    "commit-sha": "31477e4f07f3ef7bd36dfd7de44b6581c31b5099",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "underwater",
    "pr-id": "113",
    "commit-sha": "75b116461a71dcb349a2425dea0a481e739a821f",
    "pipeline": "master"
  }
]
```

File: specs/healthcheck.spec [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Model
+model:
+  rest_name: healthcheck
+  resource_name: healthchecks
+  entity_name: HealthCheck
+  package: ultros
+  group: core/monitoring
+  description: |-
+    This API allows to retrieve a generic health state of the platform.
+    A return code of different from 200 OK means the platform is not operational.
```
Comment (primalmotion) [inline specs/healthcheck.spec:10]: Typo. "Code of Different"

File: specs/healthcheck.spec [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    autogenerated: true
+    omit_empty: true
+
+  - name: status
+    description: The current health of the observed sub system.
+    type: enum
+    exposed: true
+    read_only: true
+    allowed_choices:
+    - Degraded
```
Comment (primalmotion) [inline specs/healthcheck.spec:59]: Misaligned time form. Optimal, Degraded, Error or something like that would be better

**Comment by CyrilPeponnet** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "healthcheck",
    "component": "underwater",
    "pr-id": "113",
    "commit-sha": "b3b39eabc487ab475d4a4957dfde4b186c15c20a",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "voila",
    "pr-id": "100",
    "commit-sha": "69d8c8c6bb8cd5db50e06da1d40f87a00d88f4eb",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "backend",
    "pr-id": "772",
    "commit-sha": "f279593b7a9557095a082f8b4defc846452a2667",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "apomux",
    "pr-id": "126",
    "commit-sha": "51dab71db1e64404e31fb339f9695cdf4e82d164",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "gaia",
    "pr-id": "504",
    "commit-sha": "c50f2b9222de811912695c62d5df5af1f70b19e3",
    "pipeline": "master"
  }
]
```

**Comment by CyrilPeponnet** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "healthcheck",
    "component": "gaia",
    "pr-id": "504",
    "commit-sha": "c50f2b9222de811912695c62d5df5af1f70b19e3",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "underwater",
    "pr-id": "113",
    "commit-sha": "b3b39eabc487ab475d4a4957dfde4b186c15c20a",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "voila",
    "pr-id": "100",
    "commit-sha": "69d8c8c6bb8cd5db50e06da1d40f87a00d88f4eb",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "backend",
    "pr-id": "788",
    "commit-sha": "82f5c7e4e39d61262d0df9453ba9685d56a12b7c",
    "pipeline": "master"
  },
  {
    "project": "healthcheck",
    "component": "apomux",
    "pr-id": "126",
    "commit-sha": "51dab71db1e64404e31fb339f9695cdf4e82d164",
    "pipeline": "master"
  }
]
```

**Pull request merged**